app/components/rdf-form-fields/rich-text-editor.js

```diff
@@ -1,26 +1,33 @@
 import { action } from '@ember/object';
 import { guidFor } from '@ember/object/internals';
 import { tracked } from '@glimmer/tracking';
-import { ProseParser, Schema, Selection } from '@lblod/ember-rdfa-editor';
 import {
-  block_rdfa,
-  doc,
+  inputRules,
+  ProseParser,
+  Schema,
+  Selection,
+} from '@lblod/ember-rdfa-editor';
+import {
+  blockRdfaWithConfig,
+  docWithConfig,
   hard_break,
   horizontal_rule,
-  invisible_rdfa,
+  invisibleRdfaWithConfig,
   paragraph,
-  repaired_block,
+  repairedBlockWithConfig,
   text,
-} from '@lblod/ember-rdfa-editor/nodes';
+} from '@lblod/ember-rdfa-editor/nodes'; // check
 import { inline_rdfa } from '@lblod/ember-rdfa-editor/marks';
+import { chromeHacksPlugin } from '@lblod/ember-rdfa-editor/plugins/chrome-hacks-plugin';
+import { firefoxCursorFix } from '@lblod/ember-rdfa-editor/plugins/firefox-cursor-fix';
```

Comment:
These import changes might not be needed, I just added everything I found in the example to see if it fixed the bug.